scripts/MCNPMesh.gd

```diff
@@ -0,0 +1,178 @@
+extends Spatial
+
+# arrays that contain the boundaries of voxels 
+var bounds_x = []
+var bounds_y = []
+var bounds_z = []
+
+# array that contains voxel centroids and radiation values
+# index 0 is a Vector3 of the centroid position
+# index 1 is radiation value
+# index 2 is relative error
+var radiation_map = []
+
+# maximum indices for the 3 axes and for radiation_map
+var max_index_x = 0
+var max_index_y = 0
+var max_index_z = 0
```

Comment:
If you name these something else, you can set them once during initialization and then don't need to test on `y_is_up` as often
```suggestion
var max_index_up = 0
var max_index_side = 0
```

Reply:
Unfortunately, that's not going to work because of how the index calculation works in `get_radiation()`.
`if y_is_up:
		index = (x_index * max_index_up * max_index_forward) + (y_index * max_index_forward) \
				+ z_index
	else:
		index = (x_index * max_index_forward * max_index_up) + (z_index * max_index_up) \
				+ y_index`
`y_index` and `z_index` still have to change spots due to the rigid nature of the original file having z as its smallest index.